sdcm/mgmt/common.py

```diff
@@ -25,6 +25,7 @@ def get_distro_name(distro_object):
         Distro.OEL7: "centos7",
         Distro.OEL8: "centos8",
         Distro.ROCKY8: "centos8",
+        Distro.ROCKY9: "centos8",
```

Comment:
shouldn't it be `centos9` ?

Reply:
It's just a mapping to get the manger repo

centos8 is just a key pointing to the same repo as centos7